src/main/java/de/tum/in/www1/artemis/web/rest/admin/AuditResource.java

```diff
@@ -40,7 +39,8 @@ public AuditResource(AuditEventService auditEventService) {
      * @param pageable the pagination information
      * @return the ResponseEntity with status 200 (OK) and the list of AuditEvents in body
      */
-    @GetMapping
+    @EnforceAdmin
+    @GetMapping("audits")
```

Comment:
```suggestion
    @GetMapping("audits")
    @EnforceAdmin
```
Not really important, but in all other resources the annotation order is swapped.